subworkflows/local/metagenomics_postprocessing.nf

```diff
@@ -0,0 +1,118 @@
+include { MALTEXTRACT          } from '../../modules/nf-core/maltextract/main'
+include { AMPS                 } from '../../modules/nf-core/amps/main'
+include { TAXPASTA_MERGE       } from '../../modules/nf-core/taxpasta/merge/main'
+include { TAXPASTA_STANDARDISE } from '../../modules/nf-core/taxpasta/standardise/main'
+include { MEGAN_RMA2INFO       } from '../../modules/nf-core/megan/rma2info/main'
+
+workflow METAGENOMICS_POSTPROCESSING {
+
+    take:
+    ch_postprocessing_input // different between each profiling --> postprocessing tool,
+                            // defined in metagenomics profiling subworkflow
+
+    main:
+    ch_versions      = Channel.empty()
+    ch_multiqc_files = Channel.empty()
+
+    // For MALT we have an additional step that includes maltextract+amps
+    if ( params.metagenomics_run_postprocessing && params.metagenomics_profiling_tool == 'malt' ) {
+
+        tax_list = Channel.fromPath(params.metagenomics_maltextract_taxonlist)
+        ncbi_dir = Channel.fromPath(params.metagenomics_maltextract_ncbidir)
+
+        // Malt could have been executed multiple times (group_size paramter and strandedness)
+        // We want to combine the chunks, but run MaltExtract on double and singlestranded individually
+        ch_strandedness = ch_postprocessing_input
+            .transpose()
+            .map{ meta, reads ->
```

Comment:
Are these reads now?

Reply:
updated to reflect correct name of channel values